crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__emoji.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: crates/sui/src/unit_tests/upgrade_compatibility_tests.rs
+expression: output
+---
+error[Compatibility E01007]: module missing
+  ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/missing_module_toml/emoji/Move.toml:1:2
+  │  
+1 │   😀[package]😀
```

Comment:
Since emojis are 4 bytes this helps ensure no issues with reading bytes vs reading characters in the  `missing_module_diag` I'll leave a comment so it's more clear why I think this test is useful.